src/package.xml

```diff
@@ -565,6 +565,8 @@
         <members>UTIL_IntegrationGateway_TEST</members>
         <members>UTIL_JobProgress_CTRL</members>
         <members>UTIL_JobProgress_CTRL_TEST</members>
+        <members>UTIL_Jwt</members>
```

Comment:
@voduyemi I think all those new labels need to be included in package.xml!!

Reply:
@ptewson-sfdo They've been included.